analysis_options.yaml

```diff
@@ -1,23 +1 @@
-include: package:pedantic/analysis_options.1.8.0.yaml
-
-analyzer:
-  strong-mode:
-    implicit-casts: false
-    implicit-dynamic: false
-  language:
-    strict-inference: true
-    strict-raw-types: true
-
-linter:
-  rules:
-    - annotate_overrides
-    - avoid_null_checks_in_equality_operators
-    - avoid_print
-    - await_only_futures
-    - hash_and_equals
-    - package_api_docs
-    - package_names
-    - package_prefixed_library_names
-    - prefer_final_locals
-    - prefer_void_to_null
-    - unnecessary_overrides
+include: package:lints/recommended.yaml
```

Comment:
nit: Thanks. This is actually less strict than the previous settings to a degree.

I've replaced this with `package:matan/strict.yaml`.